pkg/handlers/v1/fetchbyip_test.go

```diff
@@ -0,0 +1,123 @@
+package v1
+
+import (
+	"context"
+	"errors"
+	"testing"
+
+	gomock "github.com/golang/mock/gomock"
+	"github.com/stretchr/testify/require"
+
```

Comment:
nit: extra line here

Reply:
I thought our import stanzas followed the pattern of "stdlib, 3rd party, our own code", each separated by a blank line. Is that not the case?

Reply:
I'd be cool with that if we agree to standardize on it, but I've more commonly seen: 
stdlib
<blank line> 
everything else